### ISSUE TITLE
Generate changelogs

### ISSUE THREAD

**Issue by rubensworks** [comunica/comunica#169]
#### Issue type:

- :heavy_plus_sign: Feature request

____
#### Description:

With every new release, we should create a changelog. This could be largely generated from the git commits.

Could be part of https://github.com/semantic-release/semantic-release

**Comment by rubensworks** [comment]
I played around with various tools such as lerna-changelog and semantic-release.

Lerna-changelog seemed promising at first. It generates changelogs based on merged PRs, which we extensively do in comunica. I couldn't get it working unfortunately: https://github.com/lerna/lerna-changelog/issues/87
So maybe in the future this will become a feasible solution.

Semantic-release does not seem to work well with monorepos. Some forks have been made already, but no native support exists. I will keep an eye on this.

So in conclusion: no really good way to generate changelogs automatically at the moment. So I suggest to create them manually for now, and put this issue on hold until the tooling improves.

**Comment by rubensworks** [comment]
Created a changelog generation tool for this that makes our lives _a lot_ easier when releasing: https://github.com/rubensworks/manual-git-changelog.js
Because manual changelog creation is a pain...
